canonical-web-and-design/ubuntu.com
# Remove state from forms

## Done

- Removed `{% include "shared/forms/_state.html" %}` from forms as it seems to be not used any more and it causes errors in marketo 
- Added country, company, title fields in marketo

## QA
e.g. 
- go to /gcp/contact-us 
- submit form
- check the list in marketo

## Issue / Card

Fixes #11725 

## Screenshots
Checked the form submission in marketo after fixing it. 
![image](https://user-images.githubusercontent.com/57550290/173829628-290085c0-7d19-4aa2-9fa5-0c1e5659e5c2.png)
